### ISSUE TITLE
Interferes with my wifi somehow

### ISSUE THREAD

**Issue by nehilsaxena** [damageless/MyoTaskerPlugin#5]
I am using Moto X XT1053 with Android 4.4.4
I tried connecting MYO armband with the plugin, it connects automatically but some how i am no longer able to access net by my Wifi connection.
At first i thought it was a issue with my wifi connection by after so many hit & tries i found out that this app is causing problems.
If i disconnect MYO and Force close MYO Tasker Plugin then my wifi connectivity is restored. as soon as i open this app wifi is again gone.
P.S- my phone does not loose wifi , it shows connected but network access is gone.

**Comment by damageless** [comment]
This is an interesting bug. Have you noticed this with any other Bluetooth devices (or, more specifically, Bluetooth LE)?

**Comment by nehilsaxena** [comment]
No, i dont have any such prob with other devices or application.
